src/kernel/mconfig_dot.ml

```diff
@@ -300,7 +308,8 @@ let postprocess_config config =
     flags        = clean config.flags;
     stdlib      = config.stdlib;
     unit_name   = config.unit_name;
-    wrapping_prefix = config.wrapping_prefix;
+    unit_name_for = config.unit_name_for;
+    wrapping_prefix = None;
```

Comment:
Probably changes to `wrapping_prefix` should go in their own PR?

Reply:
Yep this was a mistake